src/libYARP_os/src/yarp/os/LogComponent.h

```diff
@@ -85,11 +85,11 @@ class YARP_os_API LogComponent
     }
 
 #ifndef NDEBUG
-#  define yCTrace(component, ...)                       yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, nullptr, component()).trace(__VA_ARGS__)
-#  define yCTraceOnce(component, ...)                   yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, YARP_ONCE_CALLBACK, component()).trace(__VA_ARGS__)
-#  define yCTraceThreadOnce(component, ...)             yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, YARP_THREADONCE_CALLBACK, component()).trace(__VA_ARGS__)
-#  define yCTraceThrottle(component, period, ...)       yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, YARP_THROTTLE_CALLBACK(period), component()).trace(__VA_ARGS__)
-#  define yCTraceThreadThrottle(component, period, ...) yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, YARP_THREADTHROTTLE_CALLBACK(period), component()).trace(__VA_ARGS__)
+#  define yCTrace(component, ...)                       yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, 0.0, nullptr, component()).trace(__VA_ARGS__)
+#  define yCTraceOnce(component, ...)                   yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, 0.0, YARP_ONCE_CALLBACK, component()).trace(__VA_ARGS__)
+#  define yCTraceThreadOnce(component, ...)             yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, 0.0, YARP_THREADONCE_CALLBACK, component()).trace(__VA_ARGS__)
+#  define yCTraceThrottle(component, period, ...)       yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, 0.0, YARP_THROTTLE_CALLBACK(period), component()).trace(__VA_ARGS__)
+#  define yCTraceThreadThrottle(component, period, ...) yarp::os::Log(__FILE__, __LINE__, __YFUNCTION__, 0.0, YARP_THREADTHROTTLE_CALLBACK(period), component()).trace(__VA_ARGS__)
```

Comment:
You should add the `CustomTime` version for logging macros with "component" as well